handlers/deploy.go

```diff
@@ -216,10 +216,24 @@ func makeDeploymentSpec(request requests.CreateFunctionRequest, existingSecrets
 							ImagePullPolicy: imagePullPolicy,
 							LivenessProbe:   probe,
 							ReadinessProbe:  probe,
+							SecurityContext: &v1.SecurityContext{
+								ReadOnlyRootFilesystem: &request.ReadOnlyRootFilesystem,
+							},
+							VolumeMounts: []v1.VolumeMount{
```

Comment:
Please create this if and only if the `request.ReadOnlyRootFilesystem` is set to true.

Reply:
Sounds valid, I will adjust this

Reply:
Well how do we achieve this, should I adjust it on my branch? And perform a rebase to have the matching History.